flexmeasures/data/scripts/clean_database.sh

```diff
@@ -4,15 +4,64 @@ MAIN_DIR=$(pwd)
 
 # function for checking database existence
 function is_database() {
- cd /tmp
- sudo -u postgres psql -lqt | cut -d \| -f 1 | grep -wq $1
- cd ${MAIN_DIR}
+  sudo -i -u postgres psql -lqt | cut -d \| -f 1 | grep -wq $1
+}
+
+# check if the user exists
+function is_user() {
+  if sudo -i -u postgres psql -tAc "SELECT 1 FROM  pg_roles WHERE rolname='$1'" | grep -q 1; then
+    echo "$1 is already available."
```

Comment:
Please add "User " before "$1", so the log becomes easier to understand. Similar in line 16.